modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -75,11 +75,52 @@ function inope_policy_review_permission() {
 }
 
 /**
+ * Implements hook_form_alter().
+ */
+function inope_policy_review_form_alter(&$form, &$form_state, $form_id) {
+
+  global $user;
+  $account = $user;
+
+  // Store argument from the autoloader in hook_menu().
+  $node = $form_state['build_info']['args'][0];
+
+  // Check that we are only altering the inope policy review form.
+  if ($form_id == 'inope_policy_review_form') {
+
+    // Query the policy review table for any records related to the current
+    // node, unapproved notes, and are from the current user.
+    $results = db_select('policy_review', 'pr')
```

Comment:
The trick about db_selects is that -- unless your query is truly dynamic, that is, the clauses and conditions are assembled at run time and vary tremendously based on interaction or context -- it is slower than db_query. I think I mentioned that in our DB stuff, but if I didn't there it is.

So -- use dbquery for selects only -- (unless your query is very dynamic).   You can find a ton of threads from Larry Garfield on this subject. I used to insist that db_select be used -- but its slow by comparison and that opinion was wrong headed.

Use the database API for everything else.